.github/workflows/scheduled.yml

```diff
@@ -144,11 +172,11 @@ jobs:
           python3 scripts/signature.py export --presto /tmp/signatures/presto_signatures_main.json
 
       - name: Save Function Signature Stash
-        if: ${{ steps.get-sig.outputs.stash-hit != 'true' }}
+        if: ${{ github.even_name == 'pull_request' && steps.get-sig.outputs.stash-hit != 'true' }}
         uses: assignUser/stash/save@v1
         with:
           path: /tmp/signatures
-          key: function-signatures
+          key: function-signatures-${{ steps.get-head.outputs.head_main }}
```

Comment:
Should it be this : ${{ steps.get-head.outputs.head_main || github.sha }} ?

Reply:
No as this is specifically only ran in PRs we will always have the right sha for main HEAD and `github.sha` would be the last commit on your PR branch but we are saving the baseline signatures here.